[Satconfig] add warning text in second slot dual VU+ tuner

Dima73 opened the pull request:
-Warning! The second slot of Vuplus DVB-S NIM(AVL6222) dual tuner does
not support Unicable/JESS type.

Dima73 committed:
[Satconfig] add warning text in second slot dual VU+ tuner

-Warning! The second slot of Vuplus DVB-S NIM(AVL6222) dual tuner does
not support Unicable/JESS type.

littlesat commented:
Why give a warning and not block configuring? It should be avoided using if tuner x then do something else.... better consider going via systeminfo as feature...

Dima73 commented:
1)I thought blocking would raise more questions than help. 
2)systeminfo = "Vuplus DVB-S NIM(AVL6222)" and /proc/stb/frontend/this slot number/rf_switch 
This is not possible and this is an exception.

eriksl commented:
I am not against applying this, but I don't understand exactly what's the problem. This AVL6222, is it a dual tuner with only one fysical input? And no way to loop them through? That would be strange, as that is the only way it could be working at all.

Dima73 commented:
![index](https://user-images.githubusercontent.com/1756274/116380606-c9048780-a81c-11eb-89f6-961b9876beb7.jpg) dual tuner for two pysical input.
tuner 1(upper)  -->work all (Unicable/JESS, DiSEqC 1.0/1.1/1.2) and etc...
tuner 2 (lower ) -->Unicable/JESS not support -->hardware limit

eriksl commented:
Interesting. I am curious what is the problem, as both Unicable and JESS use a command scheme that is very close to DiSEqC. I can imagine you'd need to use an external loop through, though.

Dima73 commented:
By the way, this tuner supports internal loop. 
https://forums.openpli.org/topic/83111-assigning-tuners-to-loop-through-from/?view=findpost&p=1338703

eriksl commented:
And DiSEqC works, SCR not?

eriksl commented:
```
...
NIM Socket 14:
	Type: DVB-S2X
	Name: Vuplus DVB-S NIM(45308X FBC)
	Frontend_Device: 14
NIM Socket 15:
	Type: DVB-S2X
	Name: Vuplus DVB-S NIM(45308X FBC)
	Frontend_Device: 15
NIM Socket 16:
	Type: DVB-S2
	Name: Vuplus DVB-S NIM(AVL6222)
	Frontend_Device: 16
	I2C_Device: 3
NIM Socket 17:
	Type: DVB-S2
	Name: Vuplus DVB-S NIM(AVL6222)
	Frontend_Device: 17
	I2C_Device: 3
```
Over here it works with JESS...

Dima73 commented:
> 
> 
> And DiSEqC works, SCR not?

yes

Dima73 commented:
I've seen similar restrictions on  DMM
https://github.com/opendreambox/enigma2/blob/master/usr/lib/enigma2/python/Components/NimManager.py#L1142

eriksl commented:
> FIXMEE no equal to for unicable .. because each tuner need a own SCR...

Their comment doesn't really help ;-). IIRC the tuner actually has two physical inputs and I connected them both. I can image it won't work with the internal loop through?

Dima73 commented:
Unicable as SCR not work in second tuner when use physical cable .
https://www.vuplus-support.org/wbb4/index.php?thread/68777-umstellung-auf-unicable/&postID=857730#post857730

Moreover, I checked it myself.

eriksl commented:
Neither https://wiki.vuplus-support.org/index.php?title=Tunerkonfiguration_Vu%2B_Duo%C2%B2_mit_einem_DVB-S2_Dual_Tuner nor https://www.vuplus-support.org/wbb4/index.php?thread/68777-umstellung-auf-unicable/&postID=857730#post857730 say it is impossible. I only understand from it that you need to make an external loop through.

eriksl commented:
So I'd agree to add a warning saying that.

Dima73 commented:
I spent a few hours after installing the Unicable LNB until I realized that this setting does not work in the lower tuner.

eriksl commented:
I think it doesn't work on an SCR LNB, but that it does work on a SRC switch (which I have).

So how about a warning, that it will probably not work on a SCR LNB, if you select SCR there?

Dima73 commented:
Just need to change the text of the warning.
For example, probably won't work with SCR LNB/SRC switch.
How it should look right (is my English equal to a Google translator)?

Dima73 commented:
unicableList = [("unicable_lnb", _("SCR (Unicable/JESS)") + " " + _("LNB")), ("unicable_matrix", _("SCR (Unicable/JESS)") + " " + _("Switch")), ("unicable_user", _("SCR (Unicable/JESS)") + " " + _("User defined"))]

Maybe...
Warning! The bottom input of this dual tuner may not support SCR LNB type.

eriksl commented:
I agree. If you change the request I'll apply it.

I propose the wording to be "Warning: the second input of this dual tuner may not support SCR LNBs" (which indeed is almost the same as you proposed).

Dima73 committed:
[Satconfig] add warning text in second input dual VU+ tuner

-Warning: the second input of Vuplus DVB-S NIM(AVL6222) dual tuner may not support SCR LNBs.

eriksl commented:
Done! Thanks dimitrij!
